spec/signin/registerandcart/register_and_cart.txt

```diff
@@ -0,0 +1,43 @@
+'Register'
+
+Verify that user can log in.
+1. Site is open.
+2. Click 'Sign in' button. 
+3. Fill 'Email address' field with valid data.
+4. Fill 'Password' field with valid data.
+5. Click 'Sign in' button.
+
+
+Verify that the email input field is validated
+1.  Click 'Sign in' button with empty  'Email address' field.
+
+Verify that the password input field is validated
+1.  Click 'Sign in' button with empty  'Password' field.
+'Register'
+
+Verify that user can`t log in with invalid email
+1.  Click 'Sign in' button with with invalid 'Email address' field.
+
+Verify that user can`t log in with invalid password
+1.  Click 'Sign in' button with invalid 'Password' field.
+
+Verify that user can restore password
+1. Site is open.
+2. Click 'Sign in' button.
+3. Click 'Forgot your password?' link.
+4. Fill 'Email address' field with valid data.
+5. Click 'Retrive password' button.
+6. Fill 'Restore password' field with valid data.
+7. Fill 'New password' field.
+8. Fill 'Repeat password' field. 
+--------------------------------------------------------------
+

```

Comment:
Signing in is other team's responsibility. You should've worked on registering user -> Create new account